README.md

```diff
@@ -137,7 +145,8 @@ The default action for each gamepad input is as follows.
 | `Right Thumbstick Horizontal Direction` | History navigation | - | `false` |
 | `Right Thumbstick Vertical Direction` | Focus on the previous/next element | `2.5` | `false` |
 
-<!-- TODO: Mention the keyboard shortcut to open the configuration panel. -->
+The configuration panel can be opened by clicking on the Gamepad Navigator pop-up icon. You can also use the keyboard
+shortcut `Ctrl + Shift + 5` to open the configuration panel.
```

Comment:
I just checked out the branch, rebuilt, and installed the plugin.  The hotkey doesn't seem to work for me.

Reply:
@the-t-in-rtf thank you for pointing this out. While I'm not completely sure as to why it isn't working, it might be possible that the hotkey is overlapping with some other feature/functionality in the recent versions of Chrome. I tried changing that to `Alt + Shift + G` and the new shortcut seems to work, at least in Ubuntu (Linux). Could you please try the new shortcut once and confirm whether it works in macOS as well? We can try other solutions and dive deeper into the problem if this doesn't work.

Reply:
`Alt + Shift + G` works.  I couldn't figure out why the other hotkey didn't.